services/notify/__init__.py

```diff
@@ -0,0 +1,152 @@
+import os
+
+from flask import current_app
+from fsd_utils import NotifyConstants
+from notifications_python_client import NotificationsAPIClient
+from notifications_python_client.errors import APIError, TokenError
+
+
+class NotificationError(Exception):
+    def __init__(
+        self,
+        message="There was a problem sending the email through GOV.UK Notify",
+    ):
+        self.message = message
+        super().__init__(self.message)
+
+
+class NotificationService:
+    MAGIC_LINK_TEMPLATE_ID = os.environ.get("MAGIC_LINK_TEMPLATE_ID", "02a6d48a-f227-4b9a-9dd7-9e0cf203c8a2")
```

Comment:
I'm not sure about having default values sprinkled through the code, does this want to live in the `envs/default` file?

To be honest I don't think we should have a default value for this at all in the code but appreciate that's what happens now - not a blocker

Reply:
I've just looked further down and realised everything is doing this fetch from env + provide a default value, this can probably be tackled as a separate piece of work but none of it feels like the role of the notification service

Reply:
I'm on the fence - happy to discuss this but also have, as you realise, intentionally left this "for the future".